apps/verticalhome/js/app.js

```diff
@@ -157,14 +204,43 @@
       this.renderGrid();
```

Comment:
I'm guessing that this onLocalized() function will need to be updated so it doesn't just localize and render the more apps grid, but also the list of pinned apps.